XenModel/Actions/Updates/DownloadAndUpdateClientAction.cs

```diff
@@ -61,7 +61,11 @@ public class DownloadAndUpdateClientAction : AsyncAction, IByteProgressAction
         private string checksum;
         private WebClient client;

```

Comment:
very nit: if you're making other changes, could you rename all private fields here using the `_variableName` convention?

This will also allow you to remove `this.` calls in the constructor
![image](https://user-images.githubusercontent.com/32554698/159442225-286900f5-5fe3-4f30-9642-3f2b698fa046.png)